insights/parsers/freeradius.py

```diff
@@ -0,0 +1,53 @@
+"""
+FreeRadius - Freeradius configuration files
+===========================================
+
+FreeRadius is a parser for the freeradius configuration files usually in
+``/etc/raddb``.
+Files ending with ``.rpmnew`` are excluded because they are backups made when
+ugprading but customer modified the configuration files, so are not used by the
+service.
+
+Because this parser reads multiple files, the files are stored as a list within
+the parser and need to be iterated through in order to find specific
+configuration files.
+
```

Comment:
Really needs examples of what the files look like and a discussion of the format they're going to be stored in.